packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.pcss

```diff
@@ -6,8 +6,11 @@ atomic-result-printable-uri button {
   color: var(--atomic-primary);
 }
 
-atomic-result-printable-uri li::after {
-  content: var(--atomic-separator);
+atomic-result-printable-uri span[part="result-printable-uri-list-separator"] svg {
```

Comment:
I wouldn't use the part here, this is a very "precise" css selector that will be hard to override from outside atomic-result